test/unit/spec/v1/PrimaryAuth_spec.js

```diff
@@ -3296,7 +3198,7 @@ Expect.describe('PrimaryAuth', function() {
     // cannot mock it because we defer to AuthJs to do set window.location.
     // On the plus side, there is an e2e test that covers this.
     // eslint-disable-next-line jasmine/no-disabled-tests
-    xit('redirects to the correct url in the social idp redirect flow');
```

Comment:
should this stay uncommented?

Reply:
Deleting this since there is already an e2e test that covers this.